### ISSUE TITLE
Word Buffer

### ISSUE THREAD

**Issue by sollaholla** [FrostCo/AdvancedProfanityFilter#536]
_Instructions: This template is written in Markdown. Click the "Preview" tab to see what it will look like when submitted. Fill in the applicable sections (you can erase the examples if you'd like)._

#### :loudspeaker: The Shortcoming
Currently, subtitles may not be accurately aligned with the actual video audio and may only appear 1-2 words after they are spoken.

#### :microscope: Describe the solution you'd like
Adding a "word buffer" that allows the profanity filter to mute even 1-2 'surrounding words' before and/or after the profanity, thereby preventing the profanity, even if it's off the audio, from being heard.

#### :triangular_ruler: Describe alternatives you've considered
--

#### :pencil: Additional context
--

**Comment by kevinjavitz** [comment]
good idea, there seems to be already the ability to filter after Unmute delay in the settings, but not unmute delay preceeding the word, which is sometimes what is still heard.
